lib/medium-admin.php

```diff
@@ -44,6 +44,7 @@ public static function init() {
     add_action("add_meta_boxes_post", array("Medium_Admin", "add_meta_boxes_post"));
 
     add_action("save_post", array("Medium_Admin", "save_post"), 10, 2);
+    add_action("publish_post", array("Medium_Admin", "save_post"), 10, 2);
```

Comment:
I'd have to test this thoroughly, but I believe I had save post tied to publish_post previously and there was some double processing which caused issues.